locales-pending/landing-premium.ftl

```diff
@@ -12,3 +12,62 @@ landing-premium-hero-image-removal-badge-label = exposures auto‑removed
 
 # This is a label underneath a big number "50" - it's an image that demos Monitor.
 landing-premium-hero-image-chart-label = fixed
+
+landing-premium-plans-heading = Choose your level of protection
+landing-premium-plans-lead = We believe in your right to privacy, so data breach monitoring is always free. For more robust protection, { -brand-monitor } also offers continuous automatic removal of your personal information.
+landing-premium-plans-table-annotation-plus = Recommended
+landing-premium-plans-table-heading-feature = Features
+# Users can click a button with this label to see a popover with a bit more detail.
+landing-premium-plans-table-feature-callout-trigger = More info
+landing-premium-plans-table-heading-free-title = { -brand-monitor }
+landing-premium-plans-table-heading-free-subtitle = Free breach alerts
+landing-premium-plans-table-heading-plus-title = { -brand-monitor } <b>Plus</b>
+landing-premium-plans-table-heading-plus-subtitle = Automatic data removal
+# Variables:
+#   $dataBrokerTotalCount (number) - number of scanned data broker sites, e.g. 190
+landing-premium-plans-table-feature-scan-label = Scan { $dataBrokerTotalCount } data broker sites that may be selling your personal info
```

Comment:
Please make this a plural string ([example](https://github.com/mozilla/blurts-server/blob/main/locales-pending/premium.ftl#L48-L51))

Reply:
Ah yeah, this number would only potentially be higher in the future, but better safe than sorry: 318357ae2